mne/stats/regression.py

```diff
@@ -134,3 +143,208 @@ def _fit_lm(data, design_matrix, names):
         mlog10_p_val[predictor] = -np.log10(p_val[predictor])
 
     return beta, stderr, t_val, p_val, mlog10_p_val
+
+
+def linear_regression_raw(raw, events, event_id=None,
+                          tmin=-.1, tmax=1,
```

Comment:
Tmin and tmax aren't for cropping, but roughly correspond to epochs tmin and tmax actually. The API here has been proposed by Alex I think. 





On Mon, Jul 20, 2015 at 9:31 AM -0700, "Eric Larson" <notifications@github.com> wrote:












In mne/stats/regression.py:
> @@ -134,3 +143,208 @@ def _fit_lm(data, design_matrix, names):
>          mlog10_p_val[predictor] = -np.log10(p_val[predictor])
>  
>      return beta, stderr, t_val, p_val, mlog10_p_val
> +
> +
> +def linear_regression_raw(raw, events, event_id=None,
> +                          tmin=-.1, tmax=1,



not sure we need tmin and tmax since it's pretty easy to crop raw instances, WDYT?



—
Reply to this email directly or view it on GitHub.